specs/helpers/setup.spec.js

```diff
@@ -0,0 +1,68 @@
+const setup = require("../../src/helpers/setup");
+const api = require("../../src/workflows/steps/index.js"); // eslint-disable-line no-unused-vars
+const getCurrentBranch = require("../../src/helpers/getCurrentBranch"); // eslint-disable-line no-unused-vars
+const filterFlowBasedOnDevelopBranch = require("../../src/helpers/filterFlowBasedOnDevelopBranch"); // eslint-disable-line no-unused-vars
+const utils = require("../../src/utils"); // eslint-disable-line no-unused-vars
+
+jest.mock("../../src/workflows/steps/index.js", () => ({
+	checkHasDevelopBranch: jest.fn(() => true)
+}));
+
+jest.mock("../../src/utils", () => ({
+	exec: jest.fn(() => Promise.resolve()),
+	getCurrentVersion: jest.fn(() => Promise.resolve("1.1.1")),
+	fileExists: jest.fn(() => true),
+	advise: jest.fn()
+}));
+
+jest.mock("../../src/helpers/filterFlowBasedOnDevelopBranch", () =>
+	jest.fn(() => ["one", "two", "three"])
+);
+jest.mock("../../src/helpers/getCurrentBranch", () =>
+	jest.fn(() => Promise.resolve("current-branch"))
+);
+
+describe("setup", () => {
+	let state;
+
+	beforeEach(() => {
+		state = {};
+	});
+
+	afterEach(() => {
```

Comment:
I think you can remove the afterEach here since the beforeEach will run before each test. You should have a fresh start each time